libdrgn/debug_info.c

```diff
@@ -4195,6 +4195,7 @@ drgn_enum_type_from_dwarf(struct drgn_debug_info *dbinfo,
 		if (err)
 			goto err;
 		compatible_type = qualified_compatible_type.type;
+		compatible_type = drgn_underlying_type(compatible_type);
```

Comment:
This is just me be very picky, but could you write this as:

```suggestion
		compatible_type =
			drgn_underlying_type(qualified_compatible_type.type);
```

I prefer assigning a variable with a single expression rather than reassigning a variable multiple times.